qemu/tests/cfg/multi_nics_hotplug.cfg

```diff
@@ -24,15 +24,25 @@
             extra_params += "-net none"
         - nic_e1000:
             no ppc64 ppc64le
+            no RHEL.8
+            no Host_RHEL.m8
```

Comment:
RHEL8 kernel don't have e1000 driver will only effect RHEL8 guest.
It is ok to boot a RHEL7 guest with e1000 device on RHEL8 host.
So, `no Host_RHEL.m8` is in-correct,  `no RHEL.8` is enough.